p2p/src/authenticated/actors/peer/actor.rs

```diff
@@ -102,14 +103,14 @@ impl<E: Spawner + Clock + ReasonablyRealtime + Rng + CryptoRng> Actor<E> {
                         content,
                     }
                 })),
-            };
+            }.encode_to_vec();
             trace!(
                 part = part,
                 total_parts = total_parts - 1,
                 peer = hex(peer),
                 "sending chunk",
             );
-            sender.send(msg).await.map_err(Error::SendFailed)?;
+            sender.send(Bytes::from(msg)).await.map_err(Error::SendFailed)?;
```

Comment:
You shouldn't need this wrapper. Use `.encode_to_vec().into()` above (like you do below).